src/Application/Cli/DeployCommand.php

```diff
@@ -93,9 +100,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $sourceDirectory = $input->getArgument('source');
         $repositoryUrl = $input->getOption('repository');
         $targetBranch = $input->getOption('branch');
+        $tempConfigRaw = $input->getOption('config');
 
         $project = new Project($sourceDirectory, getcwd().'/.couscous/generated');
 
+        $project->metadata['tempConfigRaw'] = $tempConfigRaw;
```

Comment:
`cliConfig` works, albeit a bit clunky to read. I think `tempConfigRaw` was a leftover name from when I was storing the processed array in `tempConfig` and wanted to differentiate.